DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_without_traffic.py

```diff
@@ -0,0 +1,94 @@
+import json
```

Comment:
We should not group tests based on the traffic-sending criteria. Tests should be grouped based on the feature.